src/textual/widgets/_directory_tree.py

```diff
@@ -64,19 +64,32 @@ class FileSelected(Message, bubble=True):
         `DirectoryTree` or in a parent widget in the DOM.
         """
 
-        def __init__(self, node: TreeNode[DirEntry], path: Path) -> None:
+        def __init__(
+            self, tree: DirectoryTree, node: TreeNode[DirEntry], path: Path
```

Comment:
I realise this is a bit late now, but this will have broken `DirectoryTree` as it didn't update the use of the message later on in the code.

Reply:
Thanks. Sorry for the mess-up D: